scripts/microbe_stage/configs.lua

```diff
@@ -131,6 +133,43 @@ processes = {
     },
 }
 
+-- currently only stores process capacity data for processing organelles, but can later also store material costs for organelles
+organelles = {
+    nucleus = {
+        processes = {
+            ReproductaseSynthesis = 1,
```

Comment:
No, this is a measure of how much capacity the organelle has to carry out the process